### PR TITLE
Use Alert to Show Message Prompt

### LINKED ISSUE #10: Use alert to show message prompt

### PR DESCRIPTION
**Pull request by AkhilXavier95**
Fixes #10

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Tasks/DeleteModal.jsx [hunk 1]
```diff
@@ -1,37 +1,21 @@
...
   return (
-    <Modal
+    <Alert
       isOpen
-      size="small"
-      autoHeight
-      showFooter
+      title="Delete Task"
+      message="Are you sure you want to delete the task?"
+      icon="text-red-500 ri-alarm-warning-fill ri-lg"
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Tasks/DeleteModal.jsx]: Do we need this anymore? If not, please remove it.

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@AkhilXavier95 Please make the required changes.
1. Please change the title of PR to what is fixed by the PR. In this case, it could be `Use Alert to Show Message Prompt`. 
2. Put `Fixes #IssueNumber` as the first comment, not the title.

File: app/javascript/src/components/Dashboard/Tasks/DeleteModal.jsx (hunk 1 above)
Comment (AkhilXavier95) [reply app/javascript/src/components/Dashboard/Tasks/DeleteModal.jsx]: We do not. was trying to make it the same as the Figma design.
Removed title message and icon props from Alert

**Comment by karthiknmenon** [comment]
LGTM.

**Review by karthiknmenon — APPROVED** [review]

**Pull request merged**